2020/index.md

```diff
@@ -0,0 +1,181 @@
+
+The [first NLP-OSS workshop](../2018), which was co-located with ACL 2018, was the first workshop in recent years that focused more on building quality software for NLP, open sourcing, developing useful engineering practices, and less on scientific novelty or state-of-art development. We hope that the 2nd NLP-OSS workshop could also be hosted in an *ACL conference, to be an intellectual forum to collate this type of knowledge, announce new software/features, promote the open source culture and best practices that go beyond the conferences.
+
+
+## Call for Papers
+
+We invite full papers (8 pages) or short papers (4 pages) on topics related to NLP-OSS broadly categorized into (i) software development, (ii) scientific contribution and (iii) NLP-OSS case studies.
+
+ - **Software Development**
+   - Designing and developing NLP-OSS
+   - Licensing issues in NLP-OSS
+   - Backwards compatibility and stale code in NLP-OSS
+   - Growing, maintaining and motivating an NLP-OSS community
+   - Best practices for NLP-OSS documentation and testing
+   - Contribution to NLP-OSS without coding
+   - Incentivizing OSS contributions in NLP
+   - Commercialization and Intellectual Property of NLP-OSS
+   - Defining and managing NLP-OSS project scope
+   - Issues in API design for NLP
+   - NLP-OSS software interoperability
+   - Analysis of the NLP-OSS community
+
+ - **Scientific Contribution**
+   - Surveying OSS for specific NLP task(s)
+   - Demonstration, introductions and/or tutorial of NLP-OSS
+   - Small but useful NLP-OSS
+   - NLP components in ML OSS
+   - Citations and references for NLP-OSS
+   - OSS and experiment replicability
+   - Gaps between existing NLP-OSS
+   - Task-generic vs task-specific software
+
+
+ - **Case studies**
+   - Case studies of how a specific bug is fixed or feature is added
+   - Writing wrappers for other NLP-OSS
+   - Writing open-source APIs for open data
+   - Teaching NLP with OSS
+   - NLP-OSS in the industry
+
+## Important dates
+
+The 2nd NLP-OSS workshop will be co-located with the EMNLP 2020 conference.
+
+- Paper submission: TBD
+- Paper Reviews Starts: TBD
+- Paper Reviews Due: TBD
+- Notification of Acceptance: TBD
+- Camera-Ready Version: TBD
+- Workshop: TBD
```

Comment:
Using [WMT20](http://www.statmt.org/wmt20/) dates a guide, how about these:

```
- Paper submission: 01 July 2020
- Paper Reviews Starts: 05 July 2020
- Paper Reviews Due: 05 August 2020
- Notification of Acceptance: 10 August 2020
- Camera-Ready Version: 25 August 2020
- Workshop: 11 or 12 November 2020
```

Reply:
LGTM! Will apply first, and we can keep the discussion going until we merge this PR.